packages/core/README.md

```diff
@@ -26,6 +26,8 @@ By default, the SDK is in automatic mode. You can flip the mode of the SDK using
 
     AWSXRay.enableAutomaticMode();
 
+    process.env.AWS_XRAY_MANUAL_MODE
```

Comment:
Nit: Can this line be moved to be directly beneath `AWSXRay.enableManualMode()` and also include a comment with the link to this PR for context? This will be very helpful to anyone reading the documentation, especially if they are looking for a fix for this memory leak.

Reply:
I tried something else, can you suggest a better way to present it?